packages/@okta/vuepress-site/docs/guides/deploy-to-production/overview/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Overview
+---
+You can deploy your Okta-protected app to many different hosting providers. You can also host it yourself.
+
+If you're developing a Single-Page Application (SPA), you'll need to host it on a web server and configure all paths to redirect to `index.html`. This is because your framework will handle the routing. You'll also need to force HTTPS.
+
+If you're developing a server-side application, for example in Java, .NET, or Node.js, you'll likely only need to do three things after deploying your app:
+
+1. Configure your app (or server settings) to force HTTPS.
+2. Configure your app to read your Okta settings (e.g., issuer, client ID, and client secret) from environment variable, or a secrets provider (like HashiCorp Vault).
```

Comment:
avoid latin abbreviations, use for example instead, missing "an" and last comma not necessary.

Configure your app to read your Okta settings (for example, issuer, client ID, and client secret) from an environment variable or a secrets provider (like HashiCorp Vault).